cryptography/Cargo.toml

```diff
@@ -15,9 +15,10 @@ commonware-codec = { workspace = true }
 commonware-utils = { workspace = true }
 bytes = { workspace = true }
 thiserror = { workspace = true }
-rand = { workspace = true }
+rand = { workspace = true, features = ["std_rng"] }
```

Comment:
Testing whether or not we can remove this feature: https://github.com/commonwarexyz/monorepo/pull/1544